tests/framework/extensions/clusters/config.go

```diff
@@ -0,0 +1,7 @@
+package clusters
+
+const ConfigurationFileKey = "clusters"
```

Comment:
I like the idea of this configuration but this seems extremely generic.  Could we move this config to the test suite that uses it and update the key accordingly.

Reply:
I agree, I moved the config out to the provisioning validation test suite.